submit/templates/submit/error_messages.html

```diff
@@ -0,0 +1,60 @@
+{% extends "submit/base.html" %}
+
+{% block within_content %}
+<h2 class="is-size-4">Errors and Help</h2>
+
+<h3>File Upload Errors</h3>
+  <dl>
+    <dt>File name contains unsupported characters. Renamed <code>file_name</code> to <code>new_file_name</code>. Check and modify any file paths that reference this file.</dt>
```

Comment:
Any way to do direct links to these definitions?

Reply:
Yes! I was undecided on what nomenclature we might want for them - meant to ask about that.
Do we have a list of errors with shortnames or programmatic names that make sense to use as # links? Any other suggestions for a schema for naming these? Some of the titles get long, might make difficult (if readable) link names.

Reply:
Sounds good. I will boldly make up a set of initial slug/code names, then! New commit coming soon.